docs/source/getting_started_guide/creating_a_basic_network.rst

```diff
@@ -0,0 +1,98 @@
++++++++++++++++++++++++++++++++
+Creating a basic 3-node network
++++++++++++++++++++++++++++++++
+
+In this section, we will create a three-node network.
+The three nodes are: foo, bar, and mal
+
+`foo -> bar <- mal`
+
+foo and mal are directly connected to bar with TCP connections.
+
+foo can reach mal by sending network packets through bar.
+
+#######################
+Receptor configurations
+#######################
+
+To create this, we will create three configuration files, one for each node.
+1. foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+      id: foo
+
+    - control-service:
+      service: control
+      filename: /tmp/foo.sock
+
+    - tcp-peer:
+      address: localhost:2222
+      redial: true
+
+    - log-level: debug
+    ...
+
+1. bar.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+      id: bar
+
+    - control-service:
+      service: control
+      filename: /tmp/bar.sock
+
+    - tcp-listener:
+      port: 2222
+
+    - log-level: debug
+    ...
+
+1. mal.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+      id: mal
+
+    - control-service:
+      service: control
+      filename: /tmp/mal.sock
+
+    - tcp-peer:
+      address: localhost:2222
+      redial: true
+
+    - log-level: debug
+
+    - work-command:
+      workType: echo
+      command: bash
+      params: "-c \"while read -r line; do echo $line; sleep 1; done\""
+      allowruntimeparams: true
+    ...
+
+1. Run the services in separate terminals
+
+.. code-block: bash
+
+    ./receptor --config foo.yml
+
+.. code-block: bash
+
+    ./receptor --config bar.yml
+
+.. code-block: bash
+
+    ./receptor --config mal.yml
+
+.. seealso::
+
+    :ref: `Configuring Receptor with a config file <https://ansible.readthedocs.io/projects/receptor/en/latest/user_guide/basic_usage.html#configuring-receptor-with-a-config-file>`_
```

Comment:
A :ref: doesn't need a URL if you are pointing to your own docs.   What you need is a label (aka anchor) in user_guide/basic.usage.rst above that section you want to link to.  See https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#style-links for details.  Basically, you add the label at the location you want to link to.
So for example, if you add a label in that file like  .. _config_receptor_config_file:, 
Then the ref becomes something like :ref:`config_receptor_config_file`   And sphinx makes the links work.  This way, so long as that section keeps that label, the ref will always work, no matter where that documentation section might be moved in this docset in the future.